src/styles/card.scss

```
@@ -0,0 +1,104 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 16px;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+  box-sizing: border-box;
+
+  border-radius: 5px;
+  border: 1px solid $accent-border-color;
+  background: $main-white;
```

No need of empty lines here
```suggestion
.product-card {
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  gap: 16px;
  padding: 32px 16px 16px;
  width: 200px;
  box-sizing: border-box;
  border-radius: 5px;
  border: 1px solid $accent-border-color;
  background: $main-white;
```